### PR TITLE
added api function get_account_history_operations

### PR DESCRIPTION
**Pull request by oxarbitrage**

### CODE REVIEW

File: libraries/app/api.cpp [hunk 1]
```diff
@@ -453,6 +453,37 @@ namespace graphene { namespace app {
...
     
+    vector<operation_history_object> history_api::get_account_history_operations( account_id_type account, 
+                                                                       int operation_id,
+                                                                       operation_history_id_type start, 
+                                                                       operation_history_id_type stop,
+                                                                       unsigned limit) const
+    {
+       FC_ASSERT( _app.chain_database() );
+       const auto& db = *_app.chain_database();
+       FC_ASSERT( limit <= 100 );
```
Comment (xeroc) [inline libraries/app/api.cpp]: if you move the assertion by -1 line .. you can spare instanciating the database if the limit is to high :D

**Comment by xeroc** [comment]
This PR looks good to me, however, I don't know enough of the database defaults to be 100% sure this code does what it does.
We need to be *extra* clear that this code cannot be used to segfault the node by using bad start/stop parameters.

**Comment by svk31** [comment]
To add to xeroc's comment, I'm also concerned about performance. Anyone able to comment on that aspect?

**Comment by oxarbitrage** [comment]
in regards to database defaults with stop, limit for a possible segfault. The function in that aspect is a copy of get_account_history.

File: libraries/app/api.cpp (hunk 1 above)
Comment (oxarbitrage) [inline libraries/app/api.cpp]: i kind of agree wityh this however that part of the code is an exact copy of get_account_history.

**Comment by vikramrajkumar** [comment]
This is effectively a line-by-line copy of `get_account_history`. Fixes/optimizations to the two calls can be handled in a separate issue if needed.

**Pull request merged**